### PR TITLE
Issue #27: Implement 'No recipes found'

### PR DESCRIPTION
**Pull request by alexfinnarn**

### CODE REVIEW

**Comment by alexfinnarn** [comment]
waiting on #37 to run tests on Travis.

File: components/PageRecipesAll.vue [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <template>
-  <AppSection v-if="recipes">
+  <AppSection>
     <div class="container">
-      <h3 class="title is-h3 has-text-centered"> All recipes </h3>
-      <RecipesAsCards :recipes="recipes"></RecipesAsCards>
+      <div v-if="recipes.length <= 1">
```
Comment (nyl-auster) [inline components/PageRecipesAll.vue]: why allow "1" value here ? this should be recipes.length === 0 ?

**Comment by alexfinnarn** [comment]
good point.

**Comment by nyl-auster** [comment]
could be recipes.length > 0 ( instead of length >= 1 ) but just being nitpicky here :-p

**Pull request merged**